Library/Homebrew/test/test_diagnostic.rb

```diff
@@ -204,20 +204,6 @@ def test_check_for_symlinked_cellar
     HOMEBREW_CELLAR.mkpath
   end
 
-  def test_check_for_autoconf
-    MacOS::Xcode.stubs(:installed?).returns true
```

Comment:
The test is for the doctor check rather than the method itself; I'm removing the doctor check because it's obsolete.